neptune/new/attributes_containers_table.py

```diff
@@ -0,0 +1,233 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import logging
+from datetime import datetime
+from typing import List, Dict, Optional, Union
+
+from neptune.new.exceptions import MetadataInconsistency
+from neptune.new.internal.backends.api_model import (
+    LeaderboardEntry,
+    AttributeWithProperties,
+    AttributeType,
+)
+from neptune.new.internal.backends.neptune_backend import NeptuneBackend
+from neptune.new.internal.container_type import ContainerType
+from neptune.new.internal.utils.paths import join_paths, parse_path
+
+logger = logging.getLogger(__name__)
+
+
+class AttributesContainersTableEntry:
+    def __init__(
+        self,
+        backend: NeptuneBackend,
+        container_type: ContainerType,
+        id_: str,
+        attributes: List[AttributeWithProperties],
+    ):
+        self._backend = backend
+        self._container_type = container_type
+        self._id = id_
+        self._attributes = attributes
+
+    def __getitem__(self, path: str) -> "LeaderboardHandler":
+        return LeaderboardHandler(self, path)
+
+    def get_attribute_type(self, path: str) -> AttributeType:
+        for attr in self._attributes:
+            if attr.path == path:
+                return attr.type
+        raise ValueError("Could not find {} attribute".format(path))
+
+    def get_attribute_value(self, path: str):
+        for attr in self._attributes:
+            if attr.path == path:
+                _type = attr.type
+                if _type == AttributeType.RUN_STATE:
```

Comment:
maybe extract these to static function and use it in `AttributesContainersTable.to_pandas` (with try catch etc.)

Reply:
Let's do it when we'll move all `Container` related models to subpackage, ok?